sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -99,8 +99,8 @@ class ServiceResponseError(AzureError):
 class HttpResponseError(AzureError):
     """A request was made, and a non-success status code was received from the service.
 
-    :param status_code: HttpResponse's status code
-    :type status_code: int
+    :param message: HttpResponse's error message
```

Comment:
Keep status code but change it to `ivar`, add ivar of reason and duplicate response as both ivar and param